src/components/attestering/Attestering.tsx

```diff
@@ -0,0 +1,120 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, Loader, Radio, RadioGroup } from '@navikt/ds-react';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+
+import ApiErrorAlert from '~components/apiErrorAlert/ApiErrorAlert';
+import LinkAsButton from '~components/linkAsButton/LinkAsButton';
+import { ApiResult } from '~lib/hooks';
+import { useI18n } from '~lib/i18n';
+import * as Routes from '~lib/routes';
+import { Nullable } from '~lib/types';
+import yup from '~lib/validering';
+import { UnderkjennelseGrunn } from '~types/Behandling';
+
+import messages from './attestering-nb';
+import styles from './attestering.module.less';
+import UnderkjennelsesForm from './UnderkjennelsesForm';
+
+export interface AttesterKlageFormData {
+    beslutning: Nullable<Beslutning>;
+    grunn: Nullable<UnderkjennelseGrunn>;
+    kommentar: Nullable<string>;
+}
+
+enum Beslutning {
+    IVERKSETT = 'iverksett',
+    UNDERKJENN = 'underkjenn',
+}
+
+const schema = yup.object<AttesterKlageFormData>({
```

Comment:
Litt rart at det er `AttesterKlageFormData`? 🤔

Reply:
Hehe ja, gikk litt raskt når jag tog utgångspunkt i attesteringssiden for klage. 😅  fikser